### PR TITLE
fix: CircleCi integration-test config

### PR DESCRIPTION
**Pull request by christophd**
This PR fixes some issues regarding the integration-test config on CircleCi so job is able to run the tests now.

Tests are still failing though and fixing needs to be continued in subsequent PRs

### CODE REVIEW

File: app/test/integration-test/pom.xml [hunk 1]
```diff
@@ -199,6 +199,13 @@
           <skip>true</skip>
         </configuration>
       </plugin>
+      <!-- Required only for CircleCi build as the used JVM to run the tests has limited cryptography -->
+      <plugin>
+        <artifactId>maven-enforcer-plugin</artifactId>
+        <configuration>
+          <skip>true</skip>
```
Comment (zregvart) [inline app/test/integration-test/pom.xml]: Could we try switching to [adoptopenjdk/openjdk8](https://hub.docker.com/r/adoptopenjdk/openjdk8) image in place of `openjdk:8-jdk` instead.

Comment (christophd) [reply app/test/integration-test/pom.xml]: as I am using Testcontainers in the tests I need to use the machine executor in CircleCi for this job. And this is using a ubuntu vm image with some JDK in it that has this cryptography restrictions. So not sure how to adjust this used JDK in this virtual machine.

Comment (zregvart) [reply app/test/integration-test/pom.xml]: Perhaps we could install the [AdoptOpenJDK debian package](https://adoptopenjdk.net/installation.html#linux-pkg)?

Comment (christophd) [reply app/test/integration-test/pom.xml]: Let me merge this one and tackle the cryptography afterwards so the hourly failing builds disappear for now.

**Pull request merged**